src/KOKKOS/npair_kokkos.cpp

```diff
@@ -173,7 +173,7 @@ void NPairKokkos<DeviceType,HALF_NEIGH,GHOST,TRI>::build(NeighList *list_)
   data.special_flag[2] = special_flag[2];
   data.special_flag[3] = special_flag[3];
 
-  if(list->d_neighbors.dimension_0()<nall) {
+  if(list->d_neighbors.dimension_0()<nall) { // Can this EVER be true??? - TIM 20170215
```

Comment:
I don't remember why I asked this question, but I guess it still stands... Can (list->d_neighbors.dimension_0()<nall) every be true at this point in the code?

Reply:
@timattox Good catch. No, it can never be true because list->grow() is called at the beginning of the same function.